pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWebAppTask.java

```diff
@@ -68,10 +67,17 @@ public String getEntryPoint() {
     @TaskAction
     public void buildWebapp() throws IOException, ClassNotFoundException {
         Project project = getProject();
-        PexExtension extension = ExtensionUtils.getPythonComponentExtension(project, PexExtension.class);
-        PythonExtension pythonExtension = ExtensionUtils.getPythonExtension(project);
-
-        if (extension.isFat()) {
+        PythonExtension extension = ExtensionUtils.getPythonExtension(project);
+
+        // Regardless of whether fat or thin zipapps are used, the container
+        // plugin will build the right container (i.e. .pex or .pyz).
+        // However, for thin zipapps, we need additional wrapper scripts
+        // generated (e.g. the gunicorn wrapper).
+        boolean isFat = extension.getZipapp().isFat();
+        if (isFat) {
+            // 2019-04-11(warsaw): FIXME: For now, we're still hard coding pex
+            // for the gunicorn file.  Make sure the `pex` dependency is
```

Comment:
We made sure it's installed in the plugin. We can remove that part of the comment.